powerfactory/powerfactory-model/src/main/java/com/powsybl/powerfactory/model/DataObject.java

```diff
@@ -375,6 +375,27 @@ public Optional<RealMatrix> findDoubleMatrixAttributeValue(String name) {
         return findGenericAttributeValue(name, DataAttributeType.DOUBLE_MATRIX);
     }
 
+    public Optional<RealMatrix> findAndParseDoubleMatrixAttributeValue(String name) {
```

Comment:
Understood. I will change it.

Reply:
In this version the `DataObject` dataBase is not uniform since `vectors` and `RealMatrix` have different format depending on the source of the imported case (dgs file or powerfactory dump). 

In this PR I only made a minor change in the `findDoubleMatrixAttributeValue` method to support both formats in the conversion process.

From my understanding I think that you prefer to have the same `DataObject `format for both sources. To get this I will prepare a separated PR including the necessary modifications of the `DgsParser` and `DgsReader` classes. Do you agree?

Reply:
Ok for me too